src/components/storefront/Cart/Layout/CartSkeleton.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { HStack, VStack, Skeleton, SkeletonText } from '@chakra-ui/react';
+
+const CartItemsSkeleton = ({ numItems = 3 }) => {
+  return (
+    <VStack flex={1} h="100%" spacing={8} w={{ base: '100%', md: 'auto' }}>
+      {Array(numItems)
+        .fill(0)
+        .map((_, i) => (
+          <HStack spacing={6} w="100%" key={i}>
+            <Skeleton h={{ base: '150px', md: '200px' }} w={{ base: '150px', md: '200px' }} />
```

Comment:
Maybe worthwhile to extract `{ base: '150px', md: '200px' }` as a variable so you don't have to repeat it 3x?

Reply:
yup i got lazy and can probably extract more of these styles, anyways fixed in 3dfe2ee4